Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_MegaLouse.xml

```diff
@@ -20,24 +20,31 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_MegaLouse"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>1</ArmorRating_Blunt>
+					<ArmorRating_Blunt>12</ArmorRating_Blunt>
 				</value>
 			</li>
 			
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_MegaLouse"]/statBases/ArmorRating_Sharp</xpath>
 				<value>
-					<ArmorRating_Sharp>1</ArmorRating_Sharp>
+					<ArmorRating_Sharp>6</ArmorRating_Sharp>

```

Comment:
Leave it. They're intentionally small and fragile. They're intended to be dangerous in a swarm, where a whole bunch can get in close and use their high damage attacks.